client/src/javascript/components/modals/settings-modal/AboutTab.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import ReactMarkdown from 'react-markdown';
+import AboutMarkdownPath from '../../../../../../ABOUT.md';
+
+import SettingsTab from './SettingsTab';
+
+export default class AboutTab extends SettingsTab {
+  state = {
+    about: null,
+  };
+
+  componentWillMount() {
+    fetch(AboutMarkdownPath)
```

Comment:
1. Is this going to work for users who have the `baseURI` config set in `config.template.js`? I can't remember if Webpack will handle this automatically not.
2. `fetch` is lovely but we should use `axios` for consistency — [here](https://github.com/jfurrow/flood/blob/master/client/src/javascript/actions/SettingsActions.js#L51-L53) is an example of a GET request.

Reply:
1. This path should always be correct unless the path of `client/src/javascript/components/modals/settings-modal/AboutTab.js` moves.
2. Can't use axois for non-HTTP requests?

Reply:
Webpack is treating this as a static external resource, it doesn't become part of the JS bundle. `fetch` is fetching the static resource and parsing it, so this should use `axios`.

Reply:
Makes sense now. It's actually calling it from a file.

I'm not sure if `baseURI` will affect this.